Modules/_fuzz/fuzzer.c

```diff
@@ -0,0 +1,118 @@
+/* A fuzz test for CPython.
+
+  Unusually for CPython, this is written in C++ for the benefit of linking with
```

Comment:
Scanning though this, it looks like C to me (as the filename suggests).

Reply:
Oops. Fixed.